### ISSUE TITLE
cc-addon-admin: provide visible labels

### ISSUE THREAD

**Issue by florian-sanders-cc** [CleverCloud/clever-components#558]
Input fields in the `<cc-addon-admin>` have no labels.
![image](https://user-images.githubusercontent.com/100240294/189308257-f66ba223-6709-4403-83d2-3c75fed41dbe.png)

We provided visually hidden labels for now but we actually need to add visible labels to these input fields.
These labels may be redundant with the headings on the left but there should always be a visible label **next to** the input field. The headings are too far from the input to be considered as visible labels.

**Comment by hsablonniere** [comment]
* Add them (team _in your face_)
* Maybe we could rename a few things
  * heading "Administration" => "Add-on administration"
  * heading "Add-on name" => just "Name"

If those were a single form with a single update button, we could have merge them into one block-section but here it's better like that for now I would say.

**Comment by florian-sanders-cc** [comment]
Great, thanks ! 

I'll update #535 PR to make these labels visible and change the heading translations :wink:
